client/question/services/code_evaluators/PythonPrereqCheckServiceSpec.js

```diff
@@ -493,7 +493,70 @@ describe('PythonPrereqCheckService', function() {
       ].join('\n');
       var prereqFailureType =
         PythonPrereqCheckService.detectAndGetWrongLanguageType(code);
-      expect(prereqFailureType).toEqual(null);
+
+      expect(prereqFailureType.getErrorName()).toEqual('notOp');
+      expect(prereqFailureType.getErrorLineNumber()).toEqual(
```

Comment:
Here and below: tests should not be introspecting private properties. Instead, hardcode the expected value based on the input that is given, and test that the values of getErrorLineNumber(), getErrorColumnNumber(), etc. match what you expect them to be.

Reply:
That makes sense/feels right, I just keep getting caught by the linter, complaining about 'magic numbers'. So I pass them in as local vars.